homeassistant/components/motionblinds_ble/sensor.py

```diff
@@ -122,7 +123,10 @@ def __init__(
     ) -> None:
         """Initialize the sensor entity."""
         super().__init__(
-            device, entry, entity_description, unique_id_suffix=entity_description.key
+            device,
+            entry,
+            entity_description,
+            unique_id_suffix=entity_description.key,
```

Comment:
```suggestion
            device, entry, entity_description, unique_id_suffix=entity_description.key,
```

Reply:
The above suggestion causes ruff to fail, no idea why that wasn't a problem previously